### PR TITLE
[zh] resync dns-pod-service.md

### PR DESCRIPTION
**Pull request by TinySong**
Signed-off-by: song <tinysong1226@gmail.com>

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
title: gggg[zh] resync dns-pod-service.md

What does gggg mean? typo?

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 1]
```diff
@@ -176,25 +176,25 @@ Kubernetes 会为命名端口创建 SRV 记录，这些端口是普通服务或
 其中包含 Pod 端口号和形为 `auto-generated-name.my-svc.my-namespace.svc.cluster-domain.example`
 的域名。
 
-## Pods
+## Pods {#pods}
```
Comment (Sea-n) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:179]: If it's not appeared in en content, I think it may be unnecessary.

Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md:179]: removed it.

**Comment by TinySong** [comment]
> title: gggg[zh] resync dns-pod-service.md
> 
> What does gggg mean? typo?

yes, that's a typo,  browser is stuck while using the vim plugin in the browser. (gg is a shortcut to jump the browser header). sorry for that.

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ weight: 20
 <!-- overview -->
 
 <!--
-Kubernetes creates DNS records for services and pods. You can contact
-services with consistent DNS names instead of IP addresses.
+Kubernetes creates DNS records for Services and Pods. You can contact
+Services with consistent DNS names instead of IP addresses.
 -->
 Kubernetes 为服务和 Pods 创建 DNS 记录。
 你可以使用一致的 DNS 名称而非 IP 地址来访问服务。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:22]:
```suggestion
Kubernetes 为 Service 和 Pod 创建 DNS 记录。
你可以使用一致的 DNS 名称而非 IP 地址来访问 Service。
```

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 3]
```diff
@@ -47,19 +47,19 @@ Pod's own namespace and the cluster's default domain.
 <!--
 ### Namespaces of Services 
 
-A DNS query may return different results based on the namespace of the pod making 
-it. DNS queries that don't specify a namespace are limited to the pod's 
-namespace. Access services in other namespaces by specifying it in the DNS query. 
+A DNS query may return different results based on the namespace of the Pod making 
+it. DNS queries that don't specify a namespace are limited to the Pod's 
+namespace. Access Services in other namespaces by specifying it in the DNS query.
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:52]: 英文如果拼写为 Service，中文不要翻译。下同。

File: content/zh/docs/concepts/services-networking/dns-pod-service.md [hunk 4]
```diff
@@ -15,55 +15,54 @@ weight: 20
...
 
 <!--
-Kubernetes creates DNS records for services and pods. You can contact
-services with consistent DNS names instead of IP addresses.
+Kubernetes creates DNS records for Services and Pods. You can contact
+Services with consistent DNS names instead of IP addresses.
 -->
-Kubernetes 为服务和 Pods 创建 DNS 记录。
-你可以使用一致的 DNS 名称而非 IP 地址来访问服务。
+Kubernetes 为 Services 和 Pods 创建 DNS 记录。
```
Comment (tengqm) [inline content/zh/docs/concepts/services-networking/dns-pod-service.md:21]: 中文中一般不使用复数。

Comment (TinySong) [reply content/zh/docs/concepts/services-networking/dns-pod-service.md]: 已更新

**Pull request merged**